src/pages/CreateClass/index.js

```diff
@@ -1,10 +1,37 @@
 import React from "react";
-
 import CreateClassComponent from '../../components/Class'
+import ClassesList from '../../components/Class/ClassesList'
+import Modal from '../../components/common/Modal'
+import "../../components/Class/ClassesList/styles.scss"
+
 
 function CreateClass() {
+  const [open, setOpen] = React.useState(false);
```

Comment:
Instead of `open`, We can use `[ showModel, setShowModel]`? which is more descriptive.

Reply:
done